scroll-animations-1/Overview.bs

```diff
@@ -804,6 +804,52 @@ instead of document's scrolling Element
 
 </div>  <!-- link-for-hint="ScrollTimeline" -->
 
+### The <dfn interface>CSSScrollTimelineRule</dfn> Interface ### {#the-css-scroll-timeline-rule-interface}
+
+<pre class='idl' export>
+[Exposed=Window]
+interface CSSScrollTimelineRule : CSSRule {
+    readonly attribute CSSOMString name;
+    readonly attribute CSSOMString source;
+    readonly attribute CSSOMString orientation;
+    readonly attribute CSSOMString start;
+    readonly attribute CSSOMString end;
+    readonly attribute CSSOMString scrollTimeRange;
+};
+</pre>
+
+<dl dfn-for=CSSScrollTimelineRule dfn-type=attribute>
+  <dt><dfn>name</dfn></dt>
+  <dd>
+    The name associated with the ''@scroll-timeline'' rule.
+  </dd>
+
+  <dt><dfn>source</dfn></dt>
+  <dd>
+    The 'source' descriptor associated with the ''@scroll-timeline'', or "none" if not specified.
+  </dd>
+
+  <dt><dfn>orientation</dfn></dt>
+  <dd>
+    The 'orientation' descriptor associated with the ''@scroll-timeline'', or "auto" if not specified.
+  </dd>
+
+  <dt><dfn>start</dfn></dt>
+  <dd>
+    The 'start' descriptor associated with the ''@scroll-timeline'', or "auto" if not specified.
+  </dd>
+
+  <dt><dfn>end</dfn></dt>
+  <dd>
+    The 'end' descriptor associated with the ''@scroll-timeline'', or "auto" if not specified.
+  </dd>
+
+  <dt><dfn>scrollTimeRange</dfn></dt>
+  <dd>
+    The 'scroll-time-range' descriptor associated with the ''@scroll-timeline'', or "auto" if not specified.
```

Comment:
Ops 'scroll-time-range' is a mistake in the spec that needs to be fixed. It was missed in the patch to [update the syntax](https://github.com/w3c/csswg-drafts/commit/7fd843a197f9515985b09adb5ad3018f390c44b7#")

I would be 'time-range'. But I am happy to merge this and fix all instances together.